src/components/Toaster.jsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Button from '@gen3/ui-component/dist/components/Button';
+import './Toaster.less';
+
+class Toaster extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      closed: false,
+    };
+  }
+
+  onClose = () => {
+    this.props.onCloseButton();
+    this.setState({ closed: true });
+  }
+
+  render() {
+    const closeButton = this.props.hasCloseButton &&
+    (
+      <Button
+        className='toaster__button'
+        onClick={this.onClose}
+        label={this.props.closeButtonText}
+        buttonType='secondary'
+      />
+    );
+    const actionButton = this.props.hasActionButton &&
+    (
+      <Button
+        className='toaster__button'
+        onClick={this.props.onActionButton}
+        label={this.props.actionButtonText}
+        buttonType='primary'
+        isPending={this.props.actionButtonIsPending}
+      />
+    );
+
+    return this.props.isEnabled ?
+      (
+        <div className='toaster__div'>
+          {closeButton}
+          {actionButton}
+          <div className='toaster__div-text'>
+            <div> {this.props.primaryText} </div>
+            <div> {this.props.otherText} </div>
+          </div>
+        </div>
+      )
+      :
+      null
+    ;
+  }
+}
+
+Toaster.propTypes = {
+  isEnabled: PropTypes.bool.isRequired,
+  primaryText: PropTypes.string,
+  otherText: PropTypes.string,
+  hasCloseButton: PropTypes.bool,
```

Comment:
I think 3 props for a button is too many: how about remove this `hasCloseButton` and only check `onCloseButton` is null or not for rendering. also same suggestion for action button